Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlaygroundSettings.swift

```diff
@@ -272,6 +272,10 @@ struct PaymentSheetTestPlaygroundSettings: Codable, Equatable {
         let jsonData = try! JSONEncoder().encode(self)
         return jsonData.base64EncodedString()
     }
+    
+    var base64URL: URL {
+        URL(string: "stp-paymentsheet-playground:/?\(base64Data)")!
```

Comment:
Are you missing a `/` after the scheme?

Reply:
I usually drop it for these sort of internal URLs, but I guess the spec requires it